mesh/nurbs.hpp

```diff
@@ -95,14 +95,16 @@ class NURBSPatch
 
    Array<KnotVector *> kv;
 
-   int sd, nd;
+   int sd, nd, ls;
```

Comment:
These variables with short, undescriptive names should be documented.

Reply:
Added explanation:
```
   int nd; // Number of knots in flattened structure
   int ls; // Number of variables per knot in flattaned structure
   int sd; // Stride for data access
```